doc/rule-descriptions.md

```diff
@@ -22,6 +22,7 @@
 | empty-heading | Ensures headings have discernible text | best-practice |
 | frame-title | Ensures &lt;iframe&gt; and &lt;frame&gt; elements contain a unique and non-empty title attribute | wcag2a, wcag241, section508, section508.22.i |
 | heading-order | Ensures the order of headings is semantically correct | best-practice |
+| href-no-hash | Ensures an anchors href value is not just # | best-practice |
```

Comment:
How about "Ensures that href values are valid link references to promote only using anchors as links"